examples/warehouse/domain/example.com/services.yaml

```diff
@@ -7,3 +7,5 @@ net:
       - address: logstash.example.com
         port: 5514
         protocol: tcp
+    zabbix:
+      server: zabbix.example.com
```

Comment:
Can there ever be more than one zabbix server (or proxy server) for any given agent?